.github/ISSUE_TEMPLATE/release_checklist.md

```diff
@@ -33,11 +33,11 @@ assignees: ''
 
 **After release**:
 - [ ] Update conda-forge [pygmt-feedstock](https://github.com/conda-forge/pygmt-feedstock) [Usually done automatically by conda-forge's bot]
-- [ ] Bump PyGMT version on https://github.com/GenericMappingTools/try-gmt
+- [ ] Bump PyGMT version on https://github.com/GenericMappingTools/try-gmt (after conda-forge update)
 - [ ] Announce the release on:
-  - [ ] GMT [forum](https://forum.generic-mapping-tools.org/c/news/)
+  - [ ] GMT [forum](https://forum.generic-mapping-tools.org/c/news/) (requires moderator status)
   - [ ] [Major/Minor releases only] GMT [website](https://github.com/GenericMappingTools/website) (News)
-  - [ ] [ResearchGate](https://www.researchgate.net/project/PyGMT-A-Python-interface-for-the-Generic-Mapping-Tools)
+  - [ ] [ResearchGate](https://www.researchgate.net/project/PyGMT-A-Python-interface-for-the-Generic-Mapping-Tools) (after forum announcement)
   - [ ] [Twitter](https://twitter.com/gmt_dev)
```

Comment:
Last twitter also mentions the forum annoucement. See https://twitter.com/gmt_dev/status/1406864460124463106.
```suggestion
  - [ ] [Twitter](https://twitter.com/gmt_dev) (after forum announcement)
```